### ISSUE TITLE
Pyro's hands do not seem to appear when loading a custom flamethrower model.

### ISSUE THREAD

**Issue by Kikio229** [Zabaniya001/TF2CA-weaponmodel_override#9]
It doesn't seem to be a problem with the models themselves I think...? It re-occurs no matter what model I use
Strange...
![image](https://user-images.githubusercontent.com/34814746/210453638-0e7dec90-1b08-436a-8881-3c97a2f3f59b.png)
![image](https://user-images.githubusercontent.com/34814746/210453786-92302c3d-6376-4ac4-8183-b8993ca1af53.png)

**Comment by Kikio229** [comment]
Specifically the on the Viewmodel... forgot to mention that.

**Comment by tsuza** [comment]
Yeah, this is a known issue for flamethrowers ( and the medic gun, but in that case, the issue is the beam that doesn't appear ) with this method.

In the recent TF2 updates an issue that has been bothering an older method for changing viewmodels has been fixed ( basically the arms had a weird and annoying red / blue / green lighting ), so I'd recommend using [that](https://github.com/nosoop/SM-TFCWXBaseAttributes) now ( which is actually the method you'd use for flamethrowers and medic guns ). 
I don't know if the one I specifically linked has been updated to take into account the fix, so please let me know if it works perfectly. If it doesn't, let me know so I can look at it and contact the original dev with a fix.
